frontend/test/metabase-visual/onboarding/urls.cy.spec.js

```diff
@@ -35,7 +35,7 @@ describe("visual tests > onboarding > URLs", () => {
     cy.wait("@collection-items");
     cy.wait("@collection-items");
 
-    cy.findByText("First collection");
+    cy.findAllByText("First collection");
```

Comment:
Even better and more explicit would be to scope this to the sidebar. 
```js
sidebar().within(()=> {
  cy.findByText("First collection");
  cy.findByText("Your personal collection");
})
```

And then make sure the "First collection" appears once again in `main` or (even better) in the table if you can get it by role.

That way, you will fix the failing test + add coverage for the new functionality.

Reply:
Fixed